tRecorderApi/api/models/take.py

```diff
@@ -1,7 +1,22 @@
+import json
+import json
+import os
+
+from django.contrib.contenttypes.fields import GenericRelation
 from django.db import models
 from django.utils.timezone import now
-from django.contrib.contenttypes.fields import GenericRelation
-#from .comment import Comment
+
+from ..models import book, language, chunk, anthology, version, chapter, mode, project
+
+Language = language.Language

```

Comment:
for these imports, you can do:

"from ..models import Book, Language, Chunk, ..."

which would save you from having to do these declarations immediately below

Reply:
that was my first attempt, it did not work... 

<img width="548" alt="error" src="https://user-images.githubusercontent.com/25593318/33454507-11aacdf2-d5e7-11e7-8af4-d7817971ded7.PNG">